roles/config-rh-sso/tasks/prereq.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: 'Pre-Req: Install required packages'
+  package:
+    name: '{{ item }}'
+    state: installed
+  with_items:
+  - firewalld
+  - python-firewall
+
+- name: 'Start firewalld service'
+  service:
+    name: firewalld
+    enabled: yes
+    state: started
+
+- name: 'Pre-Req: Open Firewall for Red Hat SSO'
+  firewalld:
+    port: '{{ item }}'
+    permanent: yes
+    state: enabled
+    immediate: yes
+  with_items:
```

Comment:
Please add this list of ports as a variable defined in `defaults/main.yml` so it can be overridden by the inventory if/when needed. Related to the `rh_sso_port` comment above.